CHANGELOG.md

```diff
@@ -1,4 +1,66 @@
-# dbt_zuora version.version
+# dbt_zuora v0.3.2
+This release introduces the following updates.
+
+## Bug Fixes  
+- Flipped the variable logic in `zuora__using_multicurrency` in these models so that the `*_amount` values are called by default rather than `*_amount_home_currency` when the variable isn't set: ([#22](https://github.com/fivetran/dbt_zuora/pull/22))
```

Comment:
```suggestion
- Resolved flipped logic around the `zuora__using_multicurrency` [variable](https://github.com/fivetran/dbt_zuora?tab=readme-ov-file#step-5-configure-the-multicurrency-variable-for-customers-billing-in-multiple-currencies). ([#22](https://github.com/fivetran/dbt_zuora/pull/22))
  - Ensured that, for multicurrency users, `*_amount_home_currency` fields are used and that `*_amount` values are used for single-currency users. Previously, this was erroneously flipped.
```
Just making it more readable but please adjust if this is incorrect in any way!

Reply:
same for pre-release notes

Reply:
Looks way better!